python/pypto/runtime/device_runner.py

```
@@ -0,0 +1,674 @@
+                check=False,
+                capture_output=True,
+                text=True,
+                timeout=300,
+            )
+            if result.returncode != 0:
+                # Another process may have succeeded
+                if not include_dir.exists():
+                    logger.warning(f"Failed to clone pto-isa:\n{result.stderr}")
+                    return None
+            if commit:
+                subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)